### PR TITLE
feat: LT-1084 - Record details of when a customs enquiry is being performed

### PR DESCRIPTION
**Pull request by jfaracik**
* Removed the "Is your good controlled?" question for HMRC apps
* Updated to the latest shared assets and forms
* Simplified HMRC E2E
* Fixed goods locations appearing as already filled in
* Fixed countries matrix task not showing
* Fixed third parties showing as completed on standard applications

this fucking cat always gets me to get in a good mood - Imgur

### CODE REVIEW

File: applications/helpers/validators.py [hunk 1]
```diff
@@ -25,8 +25,18 @@ def validate_delete_draft(request, pk, json):
     return {"errors": {"choice": [strings.applications.DeleteApplicationPage.DELETE_ERROR]}}, HTTPStatus.BAD_REQUEST
 
 
+def validate_goods_location_choice(_request, _pk, json):
+    choice = json.get("choice")
+
+    if choice:
+        put_application(_request, _pk, {"is_goods_departed": choice == "departed"})
```
Comment (andytaylorkainos) [inline applications/helpers/validators.py]: can you add an enum for these choices and update where appropriate please

Comment (jfaracik) [reply applications/helpers/validators.py]: Done

File: apply_for_a_licence/forms.py [hunk 2]
```diff
@@ -37,7 +36,6 @@ def opening_question():
         questions=[
             RadioButtons(
                 name="licence_type",
-                title=InitialApplicationQuestionsForms.OpeningQuestion.LABEL,
```
Comment (BenAndrew-Kainos) [inline apply_for_a_licence/forms.py]: Removing titles breaks accessibility

Comment (jfaracik) [reply apply_for_a_licence/forms.py]: I'll be fixing single component pages in a future forms update.

Comment (BenAndrew-Kainos) [reply apply_for_a_licence/forms.py]: Could we leave this until then :)

File: applications/helpers/validators.py [hunk 3]
```diff
@@ -25,8 +26,18 @@ def validate_delete_draft(request, pk, json):
     return {"errors": {"choice": [strings.applications.DeleteApplicationPage.DELETE_ERROR]}}, HTTPStatus.BAD_REQUEST
 
 
+def validate_goods_location_choice(_request, _pk, json):
```
Comment (efahy) [inline applications/helpers/validators.py]: This function `put`s the application, it doesn't just validate it - please rename

Comment (jfaracik) [reply applications/helpers/validators.py]: What would you have me rename it to?

Comment (efahy) [reply applications/helpers/validators.py]: How about `update_goods_location_choice`?  But I now realise its in `validators.py`, which doesn't seem right either

Comment (efahy) [reply applications/helpers/validators.py]: I guess you need a method in `applications/services.py`?

**Pull request merged**